index.js

```diff
@@ -62,7 +62,7 @@ module.exports = class MongooseTrailpack extends DatastoreTrailpack {
         schema.statics = model.statics
         schema.methods = model.methods
 
-        model.onSchema(schema)
+        model.onSchema(schema, this.app)
```

Comment:
Good for me

A quick note: All datastore has 2 methods: `config(app, Store)` & `schema(app Store)` that return objects.
trailpack-mongoose has this third method called onSchema for adding additionals configuration that must be a function called during bootstrap fase (essentially for adding hooks)

Why not move this into `config(app, Store)` for keeping things unified?